compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -2504,3 +2512,59 @@ fn get_apple_sdk_root(sdk_name: &str) -> Result<String, String> {
         Err(e) => Err(format!("failed to get {} SDK path: {}", sdk_name, e)),
     }
 }
+
```

Comment:
The LLVM linker requires the platform_version option. The Apple linker also supports it.